internal/oci/runtime_oci.go

```diff
@@ -878,6 +878,8 @@ func (r *runtimeOCI) AttachContainer(c *Container, inputStream io.Reader, output
 		stdinDone <- err
 	}()
 
+	close(receiveStdout)
```

Comment:
shouldn't this be done as part of the cleanup from the goroutines using these channels?

Otherwise it can cause a write to a closed channel.

Reply:
@giuseppe 
Your comment has been addressed.

Reply:
thanks, LGTM